app/javascript/crayons/MultiSelectAutocomplete/DefaultSelectionTemplate.jsx

```diff
@@ -0,0 +1,24 @@
+import { h } from 'preact';
+import { Icon, Button } from '@crayons';
```

Comment:
Refactor (blocking): Use the new Button component.

```suggestion
import { Icon, ButtonNew as Button } from '@crayons';
```